cairo/tests/utils/strategies.py

```diff
@@ -438,7 +438,17 @@ def register_type_strategies():
     st.register_type_strategy(ForwardRef("Simple"), simple)  # type: ignore
     st.register_type_strategy(ForwardRef("Extended"), extended)  # type: ignore
     st.register_type_strategy(Account, account_strategy)
-    st.register_type_strategy(Withdrawal, st.builds(Withdrawal))
+    st.register_type_strategy(
+        Withdrawal,
+        st.builds(
+            Withdrawal,
+            # Withdrawal amount is in gwei, so we need to convert it to wei
+            # thus its max value is (2**256 - 1) // (10 ** 9)
+            amount=st.integers(min_value=0, max_value=(2**256 - 1) // (10**9)).map(
+                lambda x: U256(x)
+            ),
```

Comment:
maybe a comment with the source?
With this we limit the range of data generated and U256_mul will never overflow in tests